### PR TITLE
[OSDOCS-5467]Expand all RACI tables to include relevant AWS infrastructure

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-5467
Link to docs preview:

https://58760--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @yuwang-RH and/or  @xueli181114,
   Could you please provide a QE review when you get a moment? Thanks!

File: modules/rosa-policy-shared-responsibility.adoc [hunk 1]
```diff
@@ -4,152 +4,440 @@
...
+- Use IAM tools to apply the appropriate
+permissions to AWS resources in the customer account.
+
+|Hardware/AWS global infrastructure
+|**AWS**
 
-|Virtual networking
-|Monitor cloud load balancers, subnets, and public cloud components necessary for default platform networking, and respond to alerts.
-|Monitor network traffic that is optionally configured through VPC to VPC connection, VPN connection, or Direct connection for potential issues or security threats.
+- For information regarding AWS incident and operations management, see link:https://docs.aws.amazon.com/whitepapers/latest/aws-operational-resilience/how-aws-maintains-operational-resilience-and-continuity-of-service.html#incident-management[How AWS maintains operational
```
Comment (xueli181114) [inline modules/rosa-policy-shared-responsibility.adoc:63]: This is in same content with above part, is this expected?

**Comment by xueli181114** [comment]
Comments added. Other LGTM

File: modules/rosa-policy-shared-responsibility.adoc (hunk 1 above)
Comment (mletalie) [reply modules/rosa-policy-shared-responsibility.adoc:63]: Thanks for reviewing these docs, @xueli181114. Yes, this is expected per AWS. -ML

**Review by arendej — APPROVED** [review]
LGTM

File: modules/rosa-policy-customer-responsibility.adoc [hunk 2]
```diff
@@ -7,30 +7,50 @@
...
-- Maintain responsibility for monitoring the applications run on {product-title}, including installing and operating software to gather metrics and create alerts.
+- Use the provided tools and features to link:https://docs.openshift.com/container-platform/4.12/architecture/understanding-development.html#application-types[configure and deploy]; link:https://docs.openshift.com/container-platform/4.12/applications/deployments/deployment-strategies.html[keep up to date]; link:https://docs.openshift.com/dedicated/welcome/index.html[set up resource requests and limits]; link:https://docs.openshift.com/dedicated/welcome/index.html[size the cluster to have enough resources to run apps]; link:https://docs.openshift.com/dedicated/welcome/index.html[set up permissions]; integrate with other services; link:https://docs.openshift.com/container-platform/4.5/openshift_images/images-understand.html[manage any image streams or templates that the customer deploys]; link:https://docs.openshift.com/dedicated/welcome/index.html[externally serve]; save, back up, and restore data; and otherwise manage their highly available and resilient workloads.
```
Comment (eohartman) [inline modules/rosa-policy-customer-responsibility.adoc:52]: Some of these links direct to the OSD welcome page. Is this intentional?
- set up resource requests and limits
- size the cluster to have enough resources to run apps
- integrate with other services
- externally serve

Double check the rest of the links, too, please!

**Comment by eohartman** [comment]
@mletalie Left a comment about the links, but otherwise looks good!

File: modules/rosa-policy-customer-responsibility.adoc (hunk 2 above)
Comment (mletalie) [reply modules/rosa-policy-customer-responsibility.adoc:52]: Hello @eohartman,
   Thanks for the review and notes. Yes, those links were intentionally linking to the OSD welcome page per AWS feedback. That being said, I will double (triple?) check to see if that does indeed make the most sense. Thanks!

**Comment by mletalie** [comment]
Based on feedback, removed discussed links as it was decided they did not add any overall value to the viewer, and may lead to confusion.

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**